modules/azure/networking.tf

```diff
@@ -7,12 +7,12 @@ resource "azurerm_virtual_network" "alz" {
 }
 
 resource "azurerm_subnet" "container_instances" {
-  count                                     = local.use_private_networking ? 1 : 0
-  name                                      = var.virtual_network_subnet_name_container_instances
-  resource_group_name                       = azurerm_resource_group.network[0].name
-  virtual_network_name                      = azurerm_virtual_network.alz[0].name
-  address_prefixes                          = [var.virtual_network_subnet_address_prefix_container_instances]
-  private_endpoint_network_policies_enabled = true
+  count                             = local.use_private_networking ? 1 : 0
+  name                              = var.virtual_network_subnet_name_container_instances
+  resource_group_name               = azurerm_resource_group.network[0].name
+  virtual_network_name              = azurerm_virtual_network.alz[0].name
+  address_prefixes                  = [var.virtual_network_subnet_address_prefix_container_instances]
+  private_endpoint_network_policies = "Enabled"
```

Comment:
More of a curiosity, why do we have this enabled?

Reply:
Good question. I am not sure now. I will disable and retest.